helm/cluster/values.yaml

```diff
@@ -1,3 +1,27 @@
 # Auto-generated by helm-values-gen, do not edit!
 
-{}
+connectivity:
+  network:
+    pods:
+      cidrBlocks:
+        - 100.64.0.0/12
+    services:
+      cidrBlocks:
+        - 172.31.0.0/16
+internal:
```

Comment:
We would statically set these from the `cluster-$provider` app in its Helm values like this:
```
cluster:
  internal:
    ...
```

and these would be treated as an internal API, as in internal in our apps, where the `cluster` chart exposes it to the `cluster-$provider` , but it's not meant to be set by end users. This is roughly aligned with the current behavior of `Values.internal`.